### PR TITLE
Add indexer console command output

### PR DESCRIPTION
**Pull request by brosenberger**
add additional id column to bin/magento indexer:status command in order to have that information ready

### Description (*)
in order to ease the process of the unregular event an indexer has to be reindexed by a console command on a (e.g. live-)server i want to see which indizes are not in status ready and reindex them with the second command bin/magento indexer:reindex <<indexer-id>> which is now available with the status command

### Manual testing scenarios (*)

1. connect to server / run terminal locally
2. go to magento root folder
3. call bin/magento indexer:status
4. check if according id is shown as first column (it can be checked against the bin/magento indexer:info command)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php [hunk 1]
```diff
@@ -59,15 +62,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $rows[] = $rowData;
         }
 
-        usort($rows, function ($comp1, $comp2) {
-            return strcmp($comp1['Title'], $comp2['Title']);
-        });
+        usort(
+            $rows,
+            function ($comp1, $comp2) {
```
Comment (lbajsarowicz) [inline app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php]: You can add `array` type hint to both of them

**Review by lbajsarowicz — APPROVED** [review]
Small recommendation related to Type Hints.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6302](https://jira.corp.magento.com/browse/ENGCOM-6302) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@brosenberger thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by engcom-Alfa** [comment]
@lbajsarowicz Could you review the last commit?

File: app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php (hunk 1 above)
Comment (brosenberger) [reply app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php]: added type hints within the last commit

Comment (lbajsarowicz) [reply app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php]: I'm sorry, I wasn't clear what I mean:
`function (array $comp1, array $comp2) {`

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Actually I meant the argument type declaration.

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6302](https://jira.corp.magento.com/browse/ENGCOM-6302) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed 

**Before:**
before

**After:**
after

**Pull request merged**